dev/parm/config/gefs/config.stage_ic.j2

```diff
@@ -11,7 +11,7 @@ export ICSDIR="{{ ICSDIR }}" # User provided ICSDIR; blank if not provided
 export BASE_IC="{{ BASE_IC }}" # Platform home for staged ICs
 export ENSMEM="${ENSMEM:-0}" # Number of ensemble members
 
-export STAGE_IC_YAML_TMPL="${PARMgfs}/stage/master_gefs.yaml.j2"
+export STAGE_IC_YAML_TMPL="${PARMgfs}/stage/master_gefs_RT.yaml.j2"
```

Comment:
Is this a change for all GEFS test cases or just a new one? I don't see that this yaml exists or is being added in this PR. As is, this will break current GEFS tests.

Reply:
I see `master_gefs_RT.yaml.j2` being added in #3723 now. The initial question remains though = Is this a change for all GEFS test cases?